docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,172 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
+endif::[]
+
+[[log-correlation]]
+== Log correlation
+
+
+The Elastic APM .NET agent provides integrations for popular logging frameworks, which take care of
+injecting trace ID fields into your application's log records. Currently supported logging frameworks are:
+
+- Serilog
+- NLog
+
+If your favorite logging framework is not already supported, there are two other options:
+
+* Open a feature request, or contribute code, for additional support, as described in https://github.com/elastic/apm-agent-dotnet/blob/master/CONTRIBUTING.md[CONTRIBUTING.md].
+* Manually inject trace IDs into log records, as described below in <<log-correlation-manual>>.
+
+Regardless of how you integrate APM with logging, you can use {filebeat-ref}[Filebeat] to
+send your logs to Elasticsearch, in order to correlate your traces and logs and link from
+APM to the {apm-app-ref}/xpack-logs.html[Logs app].
+
+[[serilog]]
+== Serilog
```

Comment:
This renders as a top level heading right now. <img width="372" alt="Screen Shot 2019-12-18 at 11 18 37 AM" src="https://user-images.githubusercontent.com/5618806/71116058-32d04400-2188-11ea-9f01-cf0fb9500123.png">

Let's nest instead.
```suggestion
=== Serilog
```